library/Mockery/Generator/StringManipulation/Pass/MethodDefinitionPass.php

```diff
@@ -113,8 +113,15 @@ protected function renderTypeHint(Parameter $param)
         );
         $typeHint = trim($param->getTypeHintAsString());
 
-        if (!empty($typeHint) && !in_array($typeHint, $languageTypeHints)) {
-            $typeHint = '\\'.$typeHint;
+        if (!empty($typeHint)) {
+           
+            if (!in_array($typeHint, $languageTypeHints)) {
+                $typeHint = '\\'.$typeHint;
+            }
+
+            if ($param->allowsNull()) {
```

Comment:
Will not work on 7.0 when parameter has NULL default value (would generate invalid code). I'd suggest also checking whether default value is available and is _not_ NULL.